.circleci/config.yml

```diff
@@ -649,7 +649,10 @@ jobs:
           condition:
             not:
               matches:
-                pattern: "^pull/\\d+"
```

Comment:
I think this didn't work because it is not considered a "full match pattern":

>  A full match pattern must be provided, prefix matching is not an option. Though, it is recommended to enclose a pattern in ^ and $ to avoid accidental partial matches.

-- https://circleci.com/docs/2.0/configuration-reference/#logic-statements